test/entities/swap.test.ts

```diff
@@ -0,0 +1,117 @@
+import { SwapKind } from '@/types';
+import { Swap, PathWithAmount, Token, TokenAmount } from '@/entities';
+import { weightedPoolFactory } from '@/../test/factories/entities/weightedPool';
+
+describe('Swap', () => {
+    describe('limits', () => {
+        const tokens = [
+            new Token(1, '0x177127622c4a00f3d409b75571e12cb3c8973d3c', 18),
```

Comment:
Same here about using constants in a way that's easier to read which tokens are being used for testing.